docs/management/admin/event-filters.asciidoc

```diff
@@ -0,0 +1,60 @@
+[[event-filters]]
+[chapter, role="xpack"]
+= Event filters
+
+Event filters allow admins to filter endpoint events that you do not need or want stored in {es} - for example, with a high volume. By creating event filters, you can optimize your storage in {es}. All endpoint events have the `endpoint.events.network` field.
+
+Create event filters from the Event filters page or from the Hosts page.
+
+IMPORTANT: Since an event filter blocks it from ever being streamed to {es}, be cognizant if a <<create-custom-rule, custom query rule>> exists that includes any of the same conditions as the event filter, because if so, the rule will not run. This is the expected behavior of event filters.
+
+. To create an event filter via the Hosts page:
+.. Go to *Explore* -> *Hosts*.
+.. Select the *Events* tab to view the Events table.
++
+.. Find the event to create a filter, click the *More Actions* button (*...*), then click *Add Endpoint event filter*.
++
+TIP: Since you can only create filters for endpoint events, be sure to filter the Events table to events generated by the Elastic Endpoint. +
+In the KQL search bar, enter the following query: `event.dataset : endpoint.events.network`.
++
+.. Proceed to step 3.
+. To create an event filter via the Event filters page:
+.. Go to *Manage* -> *Event filters*.
+.. Click *Add Event Filter*.
++
+The *Add event filter* flyout opens.
++
+[role="screenshot"]
+image::images/event-filter.png[]
+. Enter a name for the event filter.
+. Depending on which page you add the filter, either modify the pre-populated conditions or add new conditions that define when {elastic-sec} filters events. You can define multiple conditions with `OR` and `AND` relationships. You can also add nested conditions. In the image above, the event filter filters out events whose `event.category` field is `network`, and whose `process.executable` field is the same as the specified path.
+. Add a new comment that describes identifies the filter (optional).
+. Click *Add Endpoint Event Filter*. The new filter is added to the Event filters table.
+
+[[manage-event-filters]]
+[discrete]
+== View and manage event filters
+
+The Event filters table allows you to view and manage all endpoint event filters that have been added. To view the Event filters table, go to *Manage* -> *Event filters*. Event filters appear in reverse chronological order, with the most recently modified filter at the top. Each filter has its own "card," which displays details such as the filter name, operating system, date created, and the filter conditions.
```

Comment:
Looking at the UI in Kibana, this seems more like an "Event Filter _page_" instead of a "table," because the info isn't really organized in a table format. Is possible to change this term?

Similarly, I would suggest _section_ instead of _card_ (minus the quotation marks), because if something is a card I would expect to be able to move, shuffle, or somehow interact with it, but these appear to be just static sections of the page.

Reply:
@joepeeples the design team is discussing what to call these UI elements (e.g. `table` vs. `list`, etc.) I am currently following what's in the trusted applications topic. I also previously asked them about the usage of `card` - which is correct.